lib/usd/ui/USDImportDialogCmd.cpp

```diff
@@ -93,30 +166,55 @@ MStatus USDImportDialogCmd::doIt(const MArgList& args)
 		return MS::kSuccess;
 	}
 
+	if(argData.isFlagSet(kApplyToProxyFlag))
```

Comment:
Can you add a comment here that says when using this flag, no command object is expected.